addons/mail/static/src/models/thread/thread.js

```diff
@@ -1451,6 +1485,22 @@ function factory(dependencies) {
             return `${baseHref}#model=${this.model}&id=${this.id}`;
         }
 
+        /**
+         * Get preferred partners mentions groups ordered by priority.
+         * The first group will be used to find a partner to mention, then the second, ... until the
+         * max amount of partners is reached or the last group has been searched.
+         * @see searchLocalSuggestedPartners
+         *
+         * @private
+         * @returns {mail.partner[][]}
+         */
+        _getOrderedPreferredPartnerMentionsGroups() {
+            return [
+                this.followers.map(follower => follower.partner),
```

Comment:
This doesn't look right: some followers are channels. `undefined` should not be in list.

Reply:
The intent of this list is to be filterred then when searching for partner mention, we check at this time if partner exists.